libethcore/Farm.h

```diff
@@ -565,26 +535,32 @@ class Farm : public FarmFace
     std::string m_lastSealer;
     bool b_lastMixed = false;
 
-    std::chrono::steady_clock::time_point m_lastStart;
     uint64_t m_hashrateSmoothInterval = 30000;
```

Comment:
should be removed, also the command line option if it exists

Reply:
No command line smooth interval.
It was hardcoded